src/clr-angular/forms/datepicker/model/day.model.ts

```diff
@@ -39,6 +39,14 @@ export class DayModel {
     return new DayModel(this.year, this.month, this.date);
   }
 
+  toComparisonString(): string {
```

Comment:
Since this is public, does it need a test?

Reply:
Added one.